gensim/test/test_basemodel.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2010 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""
+Automated tests for checking transformation algorithms (the models package).
+"""
+
+import six
+
+class TestBaseTopicModel():
```

Comment:
All tests should inherit from `unittest.TestCase` (or at least `object`, if you don't want this to be a real test).